README.rst

```diff
@@ -1,31 +1,28 @@
-xonsh
-=====
+.. raw:: html
 
-.. image:: https://badges.gitter.im/xonsh/xonsh.svg
-   :alt: Join the chat at https://gitter.im/xonsh/xonsh
-   :target: https://gitter.im/xonsh/xonsh?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+    <p align="center">
+    <b>xonsh</b> is a Python-powered, cross-platform, Unix-gazing shell language and command prompt.
+    </p>
 
-.. image:: https://img.shields.io/badge/%23xonsh%3Afeneas.org-Matrix-green
-   :alt: Matrix room: #xonsh:feneas.org
-   :target: https://matrix.to/#/#xonsh:feneas.org
+    <p align="center">
+    <a href="https://xon.sh/contents.html#installation"><img src="docs/_static/xonsh4.png" alt="What is xonsh?"></a>
```

Comment:
Maybe use a full URL here, if you want it to work everywhere

Reply:
Done